rules/stylelistic.js

```diff
@@ -36,13 +36,8 @@ const rules = {
   'one-var-declaration-per-line': ['warn', 'always'],
   'padded-blocks': ['warn', 'never'],
   'quotes': ['warn', 'single'],
-  'require-jsdoc': ['warn', {
-    'require': {
-      'FunctionDeclaration': true,
-      'MethodDefinition': true,
-      'ClassDeclaration': false
-    }
-  }],
+  'valid-js': 'off',
+  'require-jsdoc': 'off',
```

Comment:
The idea is that I got is that we remove all the checks for docs. I don't think it's possible to make it require 'a doc' because then it needs a settings object and we need to specify what needs to be in the doc, which is actually just a comment...